vassal-app/src/main/java/VASSAL/counters/Immobilized.java

```diff
@@ -287,6 +287,10 @@ else if (Properties.BAND_SELECT_EVENT_FILTER.equals(key)) {
     }
   }
 
+  public boolean isCanStack() {
```

Comment:
Similarly I'm not sure we should introduce this here as a public, because it will lead to checking the stack state of the piece "incompletely". A private method for "is THIS trait preventing stacking" is fine, but for outside-this-trait purposes we should be checking the property, right?